lib/Backend/Lower.cpp

```diff
@@ -24762,21 +24767,31 @@ Lowerer::LowerDivI4(IR::Instr * instr)
     }
 #endif
 
+    Assert(instr->GetSrc2());
     if (m_func->GetJITFunctionBody()->IsWasmFunction())
     {
-        m_lowererMD.EmitInt4Instr(instr);
+        if (!m_lowererMD.GenerateFastDiv(instr))
```

Comment:
>GenerateFastDiv [](start = 25, length = 15)

Please add the stub to ARM64 as well or it will break that build.

Reply:
done. 

@LouisLaf, is the security encoding of large constants relevant and necessary for the constants generated and used in this optimization? I was wondering if it could be bypassed for this case.

Reply:
Yes, any constant directly or indirectly controllable by the user possibly needs to be encoded.
I noticed some of the shift src2 are typed as TyInt8 and some are typed as TyInt32.  They should probably all be TyInt8 for consistency.

Reply:
Thanks. yes will ensure the type consistency.